awx/main/conf.py

```diff
@@ -831,6 +831,56 @@
     category_slug='system',
 )
 
+register(
+    'AWX_CALLBACK_PROFILE',
+    field_class=fields.BooleanField,
+    label=_('Debug callback timing'),
+    default=False,
+    help_text=_('Debug callback code timing'),
+    category=('Debug'),
+    category_slug='debug',
+)
+
+register(
+    'AWX_CLEANUP_PATHS',
+    field_class=fields.BooleanField,
+    label=_('Enable or Disable tmp dir cleanup'),
+    default=True,
+    help_text=_('Enable or Disable TMP Dir cleanup'),
+    category=('Debug'),
+    category_slug='debug',
+)
+
+register(
+    'AWX_REQUEST_PROFILE',
+    field_class=fields.BooleanField,
+    label=_('Debug Web Requests'),
+    default=False,
+    help_text=_('Debug web request python timing'),
+    category=('Debug'),
+    category_slug='debug',
+)
+
+register(
+    'DEFAULT_CONTAINER_RUN_OPTIONS',
+    field_class=fields.StringListField,
+    label=_('Container Run Options'),
+    default=['--network', 'slirp4netns:enable_ipv6=true'],
+    help_text=_('List of options to pass to podman run'),
```

Comment:
I think it would be fine to suggest `--debug` in the help text, if that's the idea of the typical use.